src/System.Runtime/tests/System/Reflection/AssemblyProductAttributeTests.cs

```diff
@@ -0,0 +1,21 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Xunit;
+
+namespace System.Reflection.Tests
+{
+    public class AssemblyProductAttributeTests
+    {
+        [Theory]
+        [InlineData(null)]
+        [InlineData("")]
+        [InlineData("product")]
```

Comment:
Nit: here and in some of the other tests, can you use values that are more likely to actually be used?  For example, the original test has ".NET Core" here... that'd be better than "product".  As another example, the examples earlier that were for versions in the original tests actually used version numbers, but these tests just use "version".  If you want to also use "version", that's fine, but it'd be good to have inputs that exemplify canonical usage.